Documentation/getting-started.md

```diff
@@ -53,12 +53,24 @@ Optionally, the Media for Edge Network configuration may be set or changed progr
 
 1. Installation via [Maven](https://maven.apache.org/) & [Gradle](https://gradle.org/) is the easiest and recommended way to get the AEP SDK into your Android app. Add the Mobile Core, Edge, EdgeIdentity, and EdgeMedia extensions to your project using the app's Gradle file:
 
+    ### Kotlin
```

Comment:
can we change the header to level 4 for all the ### level.  We are doing the same for other extensions.